public/app/features/correlations/types.ts

```diff
@@ -65,12 +71,10 @@ type CorrelationBase = {
   provisioned: boolean;
   orgId?: number;
   config: CorrelationConfig;
-  type: CorrelationType;
```

Comment:
This also tripped of typescript because instead of creating the discriminate union type each of the union types inherited `type: 'query' | 'external'` which is a different thing then.